Lib/test/test_peg_generator/test_c_parser.py

```diff
@@ -91,7 +105,10 @@ def tearDown(self):
 
     def build_extension(self, grammar_source):
         grammar = parse_string(grammar_source, GrammarParser)
-        generate_parser_c_extension(grammar, Path(self.tmp_path))
+        # Because setUp() already changes the current directory to the
+        # temporary path, use a relative path here to prevent excessive
+        # path lengths when compiling.
+        generate_parser_c_extension(grammar, Path('.'), library_dir=self.library_dir)
```

Comment:
Mention that library_dir is supplied to significantly speed the test compilation up in a comment.

Reply:
These should be addressed in the latest push